src/test/kotlin/fi/metatavu/oss/api/test/functional/tests/DeviceSurveyDataTestIT.kt

```diff
@@ -1042,6 +1043,159 @@ class DeviceSurveyDataTestIT : AbstractResourceTest() {
         assertOffsetDateTimeEquals(OffsetDateTime.now(), OffsetDateTime.parse(answer2.metadata!!.createdAt))
     }
 
+    @Test
+    fun testSubmitMultiSelectAnswerWithoutAnswerOptionId() = createTestBuilder().use { testBuilder ->
```

Comment:
Maybe these tests should use mixed results where for example first 5 do not have an answer and 6-10 have

Reply:
Resolved